yantra/manager.py

```diff
@@ -81,6 +92,9 @@ def get_plugins(self):
                     # base class
                     if self.plugin_type.base_class in cls.__bases__:
                         self.register_plugin(cls)
+            except Exception as e:
```

Comment:
Are we expect errors when calling register_plugin? Or just when doing load_module? I'd try and minimize the code we are doing a catch all exceptions on.